monkestation/code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -0,0 +1,19 @@
+/obj/item/projectile/bullet/SRN_rocket
+	name = "SRN rocket"
+	desc = "Fire and step back if you hit a singularity, spatial rifts love energy"
+	icon = 'monkestation/icons/obj/guns/projectiles.dmi'
+	icon_state = "srn_rocket"
+	hitsound = "sound/effects/meteorimpact.ogg"
+	damage = 5
```

Comment:
I'd say raise it to 20 or so, it's very expensive and should cause some harm from a miss.